param_server:
  ros__parameters:
    name:
      # publish
      grid: "/decision_interface/map/grid"
      cost: "/decision_interface/map/cost"
      status: "/decision_interface/status"
      transformed_map: "/decision_interface/transformed_map"
      # subscribe
      control:
        velocity: "/decision_interface/control/velocity"
        rotation: "/decision_interface/control/rotation"
        gimbal: "/decision_interface/control/gimbal"
      lidar: "/livox/lidar"
      # other
      node: "decision_interface"
      frame:
        map: "map_link"
        lidar: "lidar_link"
    # process config
    switch:
      # livox | pointcloud2
      pointcloud_type: "livox"
      cost_map: false
      grid_map: true
      publish_transformed_cloud: true
    lidar:
      livox_frames: 1
    grid:
      resolution: 0.05
      grid_width: 10.0
      lidar_blind: 0.8
      height_wight: 120.0
      ground_height: 0.2
    filter:
      threshold: 70
